app/src/androidTest/java/com/github/orkest/ViewModel/profile/ProfileViewModelTest.kt

```diff
@@ -15,24 +15,31 @@ class ProfileViewModelTest {
     private var testUserName = "testUser"
     private var testBio = "This is my bio"
     private var testNbFollowers = 10
-    private var testNbFollowings = 2
+    private var testNbFollowings = 34
     private var testProfilePicture = 45
 
     private val newUsername = "newUsername"
     private val newBio = "My new bio"
     private val newNbFollowers = 32
-    private val newNbFollowings = 0
+    private val newNbFollowings = 65
     private val newProfilePicture = 928
 
     private lateinit var user: User
+    private lateinit var newUser: User
     private var viewModel: ProfileViewModel = ProfileViewModel(testUserName)
+    private val newViewModel: ProfileViewModel = ProfileViewModel(newUsername)
 
     @Before
     fun setUp() {
+        Constants.currentLoggedUser = testUserName
         runBlocking {
             user = User(profile = Profile(testUserName, testProfilePicture, testBio, testNbFollowers, testNbFollowings))
             viewModel.userDocument(testUserName).set(user).await()
         }
+        runBlocking {
+            newUser = User(profile = Profile(newUsername, newProfilePicture, newBio, newNbFollowers, newNbFollowings))
+            newViewModel.userDocument(newUsername).set(newUser).await()
+        }

```

Comment:
I only use runBlocking in test classes because it doesn't really matter whether it is a synchronous or asynchronous operation but it can easily be modified